services/web/server/src/simcore_service_webserver/notifications/_rabbitmq_exclusive_queue_consumers.py

```diff
@@ -90,6 +95,25 @@ async def _computational_pipeline_status_message_parser(
         rabbit_message.user_id,
         include_state=True,
     )
+    _logger.info("We got an update of pipeline status %s", rabbit_message.run_result)
+    if rabbit_message.run_result in RUNNING_STATE_COMPLETED_STATES:
```

Comment:
is this convered with current tests?

Reply:
we actually have an integration test that checks this:
services/web/server/tests/integration/01/test_computation.py